packages/cli/src/commands/create.ts

```diff
@@ -37,7 +37,12 @@ export default <CommandModule<unknown, CommandOption>>{
     },
   },
   handler: async (options) => {
-    console.log(chalk.bold(`🔥 BonFHIR 🔥`));
+    // print ASCII logo
+    console.log("       \|\\         _                    _____  _   _  ___  ____        \|\\");
```

Comment:
I would probably recommend that this is extracted into its own function in a separate file.